backend/auth.py

```diff
@@ -1,8 +1,10 @@
-from flask import request, Blueprint, render_template, redirect, url_for, request, flash
-from werkzeug.security import generate_password_hash, check_password_hash
-from models import Game, User
-from flask_login import login_user, login_required, logout_user
+from werkzeug.security import check_password_hash, generate_password_hash
```

Comment:
becuase i thought that was a standard library since i didnt need it in my requirements.txt

Reply:
Now you see why it was installed even though you didn't list it?

Reply:
if its standard library yes. but if you are saying it is not then no

Reply:
It is not in the standard library.

You should delete your venv and recreate it observing the logs and see if you can figure out why it's installed.

Reply:
i think you meant the requirements file

Reply:
so it looks like when i install flask it is a library that gets included along with a few others

Reply:
Right, your project depends on Flask, and Flask has it's own set of dependencies one of which is werkzeug

Reply:
> i think you meant the requirements file

I didn't.

Reply:
i think you meant the requirements file

I didn't.
* i first deleted the venv and then recreated it but didnt see anything so then i deleted my requirements and then added them back one by one and thats how i saw flaks connection to werzeug. (i made sure to reinstall the requiremtns while in a virtual environment.)

Reply:
Yeah by `and recreate it` I meant reinstall your dependencies into the new venv.  I'm not sure what your thought was behind deleting the requirements file.  It's a plain text file, so deleting it and putting it back has no effect.  Deleteing the venv folder has the effect of removing everything that was installed within.

Reply:
Since I was just deleting and recreating I wasn't thinking about needing to reinstall the requirements file. But obviously I would need to in order for everything to run correctly. That was a not thinking through the steps after you create your virtual environment. So thank you! I fixed the order of the imports